apps/docs/src/styles.scss

```
@@ -965,10 +977,12 @@ $doc-code-font: 'JetBrains Mono', ui-monospace, SFMono-Regular, Menlo, Monaco, C
   table {
     width: 100%;
     border-spacing: 0;
-    // Force the table to claim at least its intrinsic width. Combined with
-    // nowrap on code cells, this means the wrapper scrolls when columns
-    // can't all fit, rather than cramming each word onto its own line.
-    min-width: max-content;
+    // Use a fixed-ish layout so columns size proportionally to the table
```

Stylelint (17.9.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)